_data/meltano/utilities/dbt-duckdb/jwills.yml

```
@@ -86,6 +86,7 @@ settings:
   value: duckdb
 - label: Projects Directory
   name: project_dir
+  env: DBT_PROJECT_DIR
```

Reorder to make the linter happy.

```suggestion
- env: DBT_PROJECT_DIR
  label: Projects Directory
  name: project_dir  
```